mysql-test/suite/innodb/r/innodb-page_compression_none.result

```diff
@@ -0,0 +1,20 @@
+#
+# MDEV-30825 innodb_compression_algorithm=0 (none) increments Innodb_num_pages_page_compression_error
+#
+SET GLOBAL innodb_compression_algorithm=0, innodb_checksum_algorithm=strict_full_crc32, innodb_buffer_pool_size=8*1024*1024;
```

Comment:
This is failing massively, for some reason:
```
CURRENT_TEST: innodb.innodb-page_compression_none
mysqltest: At line 8: query 'SET GLOBAL innodb_compression_algorithm=0, innodb_checksum_algorithm=strict_full_crc32, innodb_buffer_pool_size=8*1024*1024' failed: 1231: Variable 'innodb_buffer_pool_size' can't be set to the value of '8388608'
```
Wasn’t it so that in debug builds there is some extra flag that needs to be enabled to allow buffer pool resizing? Why does this test need to resize the buffer pool at all? Could the buffer pool size be specified in a `.opt` file?